openassessment/xblock/static/js/src/studio/oa_edit_fields.js

```diff
@@ -152,8 +152,8 @@ OpenAssessment.ToggleControl.prototype = {
     },
 
     show: function() {
-        this.shownSection.removeClass('is--hidden');
         this.hiddenSection.addClass('is--hidden');
+        this.shownSection.removeClass('is--hidden');
```

Comment:
By re-ordering this so that we always remove `is--hidden` last, it's possible to use the same element for `shownSection` and `hiddenSection`, to keep an element from changing when using a ToggleControl.